tests/integration/validation.py

```diff
@@ -1,20 +1,26 @@
 import json
-from typing import Union
+from typing import Any, Dict, Optional, Union
 
 import gnmi.proto
 import gnmi.proto.legacy
 
 
 def validate_default_interfaces_get(
-    response: Union[gnmi.proto.GetResponse, gnmi.proto.legacy.GetResponse]
+    response: Union[gnmi.proto.GetResponse, gnmi.proto.legacy.GetResponse],
+    value: Optional[Union[str, Dict[str, Any]]] = None,
 ):
     assert len(response.notification) == 1
 
     notification = response.notification.pop()
     assert len(notification.update) == 1
 
     update = notification.update.pop()
-    assert update.val.json_val
-    assert json.loads(update.val.json_val.decode("utf-8")) == {
-        "interface": {"admin": {"config": {"name": "admin"}, "name": "admin"}}
-    }
+
+    if isinstance(value, str):
+        assert update.val.string_val == value
+    else:
+        assert update.val.json_val
+        assert json.loads(update.val.json_val.decode("utf-8")) == (
+            value
+            or {"interface": {"admin": {"config": {"name": "admin"}, "name": "admin"}}}
```

Comment:
Whats the purpose of the `or` here?

Reply:
Ah I see, this comes from the previous intention of this method. I would rather suggest to split these verify methods to not have a default value here.

Reply:
Good idea.